package.json

```diff
@@ -25,35 +25,28 @@
         "ocamlfind",
         "sexplib"
       ]
-    },
-    "buildsInSource": "_build"
+    }
   },
   "dependencies": {
-    "@opam/alcotest": "0.8.3",
-    "@opam/astring": "0.8.3",
-    "@opam/bisect_ppx": "1.3.4",
-    "@opam/bos": "0.2.0",
-    "@opam/cmdliner": "1.0.2",
-    "@opam/cppo": "1.6.5",
-    "@opam/dune": "1.3.0",
-    "@opam/fpath": "0.7.2",
-    "@opam/js-build-tools": "113.33.04",
-    "@opam/lambdasoup": "0.6.3",
-    "@opam/markup": "*",
-    "@opam/oasis": "0.4.11",
-    "@opam/ocamlfind": "1.8.0",
-    "@opam/result": "1.3",
-    "@opam/sexplib": "*",
-    "@opam/tyxml": "4.3.0",
-    "ocaml": "~4.2.0"
-  },
-  "resolutions": {
-    "@opam/markup": "aantron/markup.ml:markup.opam#9f8e77",
-    "@opam/sexplib": "bryphe/sexplib#548f94d"
```

Comment:
You'll need this resolution to get a `sexplib` version that builds with `4.02` afaik

Reply:
There are 2 issues here, first is that it doesn't point to the opam file so it can't be resolved with esy. The second issue is that when you point to the opam file it doesn't contain metadata so it fails anyway.